Source/Dafny/DafnyPipeline.csproj

```diff
@@ -139,10 +139,10 @@
     <Reference Include="BoogieParserHelper">
       <HintPath>..\..\..\boogie\Binaries\BoogieParserHelper.dll</HintPath>
     </Reference>
-    <PackageReference Include="Mono.Cecil" Version="0.11.1" />
+    <PackageReference Include="Mono.Cecil" Version="0.11.2" />
     <Reference Include="System" />
     <Reference Include="System.Core">
-      <RequiredTargetFramework>3.5</RequiredTargetFramework>
+      <RequiredTargetFramework>4.8</RequiredTargetFramework>
```

Comment:
For context, these were the last changes I made before the build consistently succeeded: https://github.com/dafny-lang/dafny/pull/579/commits/ff551a10c6b29124d01e2e49196da9e348274cee